main-site/components/meet-the-team/mobile-team-column/MobileTeamColumn.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import Arrow from '../../../../shared-ui/components/arrow/Arrow';
+import {
+  MobileTeamColumnProps,
+  TeamColumnInfo
+} from '../../../../shared-ui/lib/types';
+import { getLeftOrRight } from '../../../lib/utils';
+import { StyledHeadshot, StyledLabel } from '../team-column/TeamColumn.styles';
+import { StyledMobileTeamColumn, LabelArrowContainer, MobileImageContainer } from './MobileTeamColumn.styles';
+
+const MobileTeamColumn: React.FC<MobileTeamColumnProps> = ({
+  listOfColumnInfo
+}) => {
+  const [curColumn, setCurColumn] = useState<TeamColumnInfo>(
+    listOfColumnInfo[0]
+  );
+  const listOfPictures: string[] = curColumn.listOfPictures;
+  return (
+    <StyledMobileTeamColumn>
+      <LabelArrowContainer>
+        <Arrow
+          left
+          onClick={(): void =>
+            setCurColumn(
+              getLeftOrRight('left', listOfColumnInfo, curColumn)
```

Comment:
Im not sure if when calling methods that utilize generic type, we should parametrize it like `getLeftOrRight<TeamColumnInfo>()` or leave it as is. I would clarify with @jzhzhang66